lib/foreman_bootdisk/engine.rb

```diff
@@ -31,11 +31,11 @@ class Engine < ::Rails::Engine
         requires_foreman '>= 1.9'
 
         security_block :bootdisk do |map|
-          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :help],
+          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :subnet, :help],
                                           :'foreman_bootdisk/api/v2/disks' => [:generic, :host]}
         end
 
-        role "Boot disk access", [:download_bootdisk]
+        role "Boot disk access", [:download_bootdisk, :view_hosts, :view_subnets]
```

Comment:
Ok this is chicken-egg problem. I think we need to move roles from engine to seeds and adopt the foreman-core approach.